### ISSUE TITLE
LineGraph uses unsafe methods, producing warnings

### ISSUE THREAD

**Issue by s100** [carbon-design-system/carbon-addons-data-viz-react#144]
# Detailed description

> Is this issue related to a specific component?

`<LineGraph>`

> What did you expect to happen? What happened instead? What would you like to see changed?

I expected to be able to use `<LineGraph>` without warnings appearing in my console. However, I see two warnings about the use of unsafe methods `componentWillReceiveProps` and `componentWillUpdate`:

```
Warning: componentWillReceiveProps has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* If you're updating state whenever props change, refactor your code to use memoization techniques or move it to static getDerivedStateFromProps. Learn more at: https://fb.me/react-derived-state
* Rename componentWillReceiveProps to UNSAFE_componentWillReceiveProps to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: LineGraph
```

and

```
Warning: componentWillUpdate has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move data fetching code or side effects to componentDidUpdate.
* Rename componentWillUpdate to UNSAFE_componentWillUpdate to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: LineGraph
```

> What browser are you working in?

Chrome 76.0.3809.100 64-bit on Windows 10

> What version of the Carbon Design System are you using?

This is `carbon-addons-data-viz-react@1.22.0`, the latest available at the time of writing.

## Steps to reproduce the issue

Basically just inspecting the code of `<LineGraph>` will demonstrate this issue!

## Additional information

This is with React 16.9.0.

## Add labels
Please choose the appropriate label(s) from our existing label list to ensure that your issue is properly categorized. This will help us to better understand and address your issue.
